public/app/percona/add-instance/components/AzureDiscovery/Discovery.test.tsx

```diff
@@ -8,8 +8,10 @@ describe('Discovery instance:: ', () => {
 
     const root = mount(<Discovery selectInstance={selectInstance} />);
 
-    expect(root.find('input[data-qa="aws_access_key-text-input"]').length).toBe(1);
-    expect(root.find('input[data-qa="aws_secret_key-password-input"]').length).toBe(1);
+    expect(root.find('input[data-qa="azure_client_id-text-input"]').length).toBe(1);
```

Comment:
Maybe use `dataQa` function to reduce noise??

Reply:
👍